appengine_config.py

```diff
@@ -68,48 +68,3 @@ def save(self):
 
     app = recording.appstats_wsgi_middleware(app)
     return app
```

Comment:
Is [AppStats](https://cloud.google.com/appengine/docs/standard/python/tools/appstats) supported in Python 3? If not, let's just get rid of it from here as well and have this PR be more about dropping the entire appengine_config.py file. (Make sure to delete other places which invoke AppStats too -- see the link in the first sentence of this comment.)